### ISSUE TITLE
Encourage using long parameter options and mnemonic tags in our style guide

### ISSUE THREAD

**Issue by reinhart1010** [tldr-pages/tldr#8304]
There's another two writing rules, undocumented on our style guide which we have also practiced instead in reviewing pull requests for new pages:

1. Prefer long parameter formats (e.g. `--verbose` instead of `-v`) as possible,
2. When the documented program does not include long parameter formats, documenters may use **parameter mnemonic tags** (e.g. `[v]erbose`)

Which I personally liked, and in order to standardize how we should treat these cases we should put them in our style guide.

This is because most of our documented commands practiced at least one of these conventions:

1. The convention deriving from DOS where all parameters are used using forward slashes, such as `cd {{path/to/directory}} /d` to switch directories.
2. The Unix-like convention as accepted by many major, cross-platform apps, such as `firefox --profile profile1 and git --help`.
4. The PowerShell convention where long parameters are also using single dash instead of 2 in Unix, like `Invoke-WebRequest -Method Post -Body @{ name='bob' } http://example.com/form`

Where many (1)-type commands may not introduce long parameter formats, as discussed on #8201, while some (2)-type commands also don't introduce long parameters as well, like in #8200. Here are some examples for the case of `slmgr.vbs` (#8201)

```
- Display the current license's e[xp]i[r]ation date and time:

`slmgr /xpr`

- [c]lears the current license's [p]roduct [k]e[y] from the Windows Registry. This will not deactivate or uninstall the current license, but prevents the key from being stolen by malicious programs in the future:

`slmgr /cpky`
```

Mnemonic tags aren't mandatory for translations as well. However, when including these tags into translations, make sure that the the tags do correspond to the same terms as in the English version or the original language where these short parameters correspond to the terms of the language (e.g. `[c]` for `[c]arregar` in Portuguese-based programs).

For example, in #8298,

```
- Hentikan hak akses terhadap [g]rup untuk mengeksekusi suatu file:

`chmod g-x {{jalan/menuju/file}}`
```

**Comment by marchersimon** [comment]
We actually have documented that we prefer long options in [CONTRIBUTING.md](https://github.com/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines). But we should definitively move this to the style guide.

**Comment by EmilySeville7cfg** [comment]
Long portable options are required now in style guide. Mnemonics are gonna to be allowed for everything, check out my PR. :)

**Comment by vitorhcl** [comment]
Fixed by #11619
